code/datums/holocall.dm

```diff
@@ -32,20 +32,29 @@
 	var/datum/action/innate/end_holocall/hangup	//hangup action
 
 	var/call_start_time
+	var/head_call = FALSE //calls from a head of staff autoconnect, if the recieving pad is not secure.
 
 //creates a holocall made by `caller` from `calling_pad` to `callees`
-/datum/holocall/New(mob/living/caller, obj/machinery/holopad/calling_pad, list/callees)
+/datum/holocall/New(mob/living/caller, obj/machinery/holopad/calling_pad, list/callees, var/elevated_access = FALSE)
 	call_start_time = world.time
 	user = caller
 	calling_pad.outgoing_call = src
 	calling_holopad = calling_pad
+	head_call = elevated_access
 	dialed_holopads = list()
 
 	for(var/I in callees)
 		var/obj/machinery/holopad/H = I
 		if(!QDELETED(H) && H.is_operational())
 			dialed_holopads += H
-			H.say("Incoming call.")
+			if(head_call)

```

Comment:
head_call is only used here, just use elevated_access from the proc arguments directly

Reply:
Line 385 of hologram.dm checks if head_call is set on the datum, so we'd need the head_call var either way.

Reply:
Ah, gotcha